php/Terminus/Models/Environment.php

```diff
@@ -140,6 +162,36 @@ public function cloneFiles($to_env) {
   }
 
   /**
+   * Commits changes to code
+   *
+   * @param [string] $commit Should be the commit message to use if committing
+   *   on server changes
+   * @return [array] $data['data']
+   */
+  public function commitChanges($commit = null) {
+    ob_start();
+    passthru('git config --global user.email');
+    $git_email = ob_get_clean();
+    ob_start();
+    passthru('git config --global user.name');
+    $git_user = ob_get_clean();
```

Comment:
I'm not sure. That makes the behavior different from the dashboard (which presumably uses the email/display name on the account). I don't know if that's a good thing or a bad thing. 

Also we need to make sure this doesn't error out if git is not installed on the users workstation.

If we are keeping this, you should be able to simplify it by using:

    $git_email = system('git config --global user.email');
    $git_user = system('git config --global user.name');

instead of `passthru` and the output buffer.